src/Illuminate/Auth/Middleware/EnsureEmailIsVerified.php

```diff
@@ -23,7 +23,7 @@ public function handle($request, Closure $next, $redirectToRoute = null)
             ! $request->user()->hasVerifiedEmail())) {
             return $request->expectsJson()
                     ? abort(403, 'Your email address is not verified.')
-                    : Redirect::route($redirectToRoute ?: 'verification.notice');
+                    : Redirect::guest($redirectToRoute ?: route('verification.notice'));
```

Comment:
The `route` function doesn't exist, so we can't call it (it's only defined within the foundation namespace).

Reply:
But the original code used `route` already, this package is considered to be used with Laravel, so I think this is not a problem.